brainstormforce/astra
# AST-2294 - fix: list bullets block misaligned with page container.

### Description

The bullets of list items are not following the content width. Its getting out of area.
### Screenshots

https://d.pr/i/N41dEP

### Types of changes


- Fix: list bullets block misaligned with page container.



### How has this been tested?

- Install Astra.
- Create a page.
- Add a list bullets block.
- Check frontend.

### Checklist:
